src/Language/PureScript/Publish.hs

```diff
@@ -128,11 +130,23 @@ catchLeft a f = either f pure a
 
 preparePackage' :: PublishOptions -> PrepareM D.UploadedPackage
 preparePackage' opts = do
-  unlessM (liftIO (doesFileExist (publishManifestFile opts))) (userError PackageManifestNotFound)
   checkCleanWorkingTree opts
 
-  pkgMeta <- liftIO (Bower.decodeFile (publishManifestFile opts))
-               >>= flip catchLeft (userError . CouldntDecodePackageManifest)
```

Comment:
Since `Bower.decodeFile` is no longer being used (i.e. Bower.asPackageMeta` is used instead), should `Bower.decodeFile` be removed entirely? Or are there other places where it's still used?

Reply:
That function comes from the https://hackage.haskell.org/package/bower-json package, so there's nothing to remove on our side.